doc/api.md

```diff
@@ -90,8 +90,9 @@ correspondence between the API resource fields and the same fields in the databa
 | `attributes`          | `attributes`         | list of common active attributes of the device. For each attribute, its `name` and `type` must be provided, additional `metadata` is optional.                                                                                                                            |
```

Comment:
I'd suggest to add the mixed mode to the NGSI-LD feature sublist in CHANGES_NEXT_RELEASE. I mean:

```
Add basic NGSI-LD support as experimental feature (#842)
...
- Commands
- Mixed mode (based in ngsiVersion field in the provisioning API)
```

Reply:
Fixed 3dee480